app/views/programmes/patients.html.erb

```diff
@@ -22,6 +22,8 @@
           form: @form,
           url: patients_programme_path(@programme),
           programme_statuses: Patient::VaccinationStatus.statuses.keys,
+          consent_statuses: Patient::ConsentStatus.statuses.keys,
+          triage_statuses: %w[safe_to_vaccinate do_not_vaccinate delay_vaccination required],
```

Comment:
I copied this from `app/views/programmes/patients.html.erb`. Was wondering if we could put it somewhere and share it between the two partials. Any ideas or just leave it for now?